sql-plugins/src/test/java/io/cdap/pipeline/sql/plugins/bigquery/BigQuerySinkTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.pipeline.sql.plugins.bigquery;
+
+import io.cdap.pipeline.sql.api.template.tables.AbstractTableInfo;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BigQuerySinkTest {
+  @Test(expected = IllegalArgumentException.class)
+  public void testNullProject() {
+    BigQuerySQLSink.BigQuerySQLSinkConfig config = new BigQuerySQLSink.BigQuerySQLSinkConfig(null, "a",
```

Comment:
similar comment for these tests, these types of things should be testable by calling methods on the config object directly rather than indirectly through the sink

Reply:
Went ahead and removed some trivial nullity and empty string checks.